packages/react/src/hooks/useTemplateModule.ts

```diff
@@ -25,16 +26,17 @@ export const useTemplateModule = (
             ? state.preferences.kpiTokenTemplateBaseURL
             : state.preferences.oracleTemplateBaseURL
     );
+    const ipfsGatewayURL = useIPFSGatewayURL();
 
     const { baseUrl, entry } = useMemo(() => {
         if (!template) return {};
         return {
             baseUrl:
                 customBaseURL ||
-                `${IPFSService.gateway}/ipfs/${template.specification.cid}`,
+                `${ipfsGatewayURL}/ipfs/${template.specification.cid}`,
```

Comment:
same, a function that given the `ipfsGatewayURL` and  `CID` return the url would be great